netbox/templates/dcim/device/interfaces.html

```diff
@@ -30,3 +30,29 @@
         </div>
     {% endif %}
 {% endblock bulk_extra_controls %}
+
+{% block head %}
+    {{ block.super }}
+    <style>
+        {% for status, color in table.Meta.cable_status_styles %}
+            tr[data-cable-status={{ status }}] {
+                background-color: var(--nbx-color-{{ color }}-a15);
+            }
+        {% endfor %}
```

Comment:
I don't think we need this, the row should not change depending on cable_status color beyond either green or blue.  Any other changes should be done in the cable form which do not need to be dynamically updated.